core/domain/email_domain.py

```diff
@@ -70,3 +70,37 @@ def thread_id(self):
         """
         return '.'.join(
             [self.entity_type, self.entity_id, self.id.split('.')[3]])
+
+
+class SendEmailInfo(python_utils.OBJECT):
+    """The information needed to send an email.
+    Attributes:
+        recipient_id: str. The user ID of the email recipient.
+        sender_id: str. The user ID of the sender.
+        intent: str. The intent string for the email, i.e. the purpose/type.
+        email_subject: str. The subject of the email.
+        email_html_body: str. The body (message) of the email.
+        sender_email: str. The sender's email address.
+        recipient_email: str. The recipient's email address.
+        bcc_admin: bool. Whether to send a copy of the email to the admin's
+            email address.
+        sender_name: str|None. The name to be shown in the "sender" field of
```

Comment:
Suggest defining how the 'None' case is handled (e.g. what is the sender in that case?)